src/system/settings.ts

```diff
@@ -1,26 +1,73 @@
-export function registerSettings() {
-    game.settings!.register('cosmere-rpg', 'firstTimeWorldCreation', {
+import { SYSTEM_ID } from './constants';
+
+/**
+ * Index of identifiers for system settings.
+ */
+export const SETTINGS = {
+    INTERNAL_FIRST_CREATION: 'firstTimeWorldCreation',
+    INTERNAL_LATEST_VERSION: 'latestVersion',
+    ITEM_SHEET_SIDE_TABS: 'itemSheetSideTabs',
+    ROLL_SKIP_DIALOG_DEFAULT: 'skipRollDialogByDefault',
+};
```

Comment:
suggest making this `as const`

Reply:
actually, could you structure it something like:
```
{
    internal: {
        FIRST_CREATION: '...',
        ...
    },
    items: {
        SHEET_SIDE_TABS: '...',
        .... ,
        defaults: {
            side_tabs: false
        }
    },
    rolls: {
        SKIP_SOMETHING: '...',
    }
}
```
Hopefully this could make the looping script even more repeatable?